ocs_ci/deployment/helpers/tnf_validation.py

```
@@ -0,0 +1,379 @@
+        "G": 1,
+        "T": 1024,
+        "P": 1024 * 1024,
+    }
+
+    for unit, multiplier in multipliers.items():
+        if unit in size_str:
+            try:
+                value = float(size_str.replace(unit, "").strip())
+                return value * multiplier
+            except ValueError:
+                logger.warning(f"Failed to parse size string: {size_str}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)